src/notification_service/sender/ses.py

```diff
@@ -0,0 +1,151 @@
+import boto3
+import botocore
+from email.message import Message as BaseEmailMessage
+from email.mime.base import MIMEBase
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from typing import List
+
+from .base import Sender
+from ..exceptions import (
+    AwsSesSenderException,
+    AwsSesUserNotVerifiedException,
+    AwsSesNoRelevantRecipientsException,
+)
+from ..messages import Message, File
+
+
+class AwsSesSender(Sender):
+    """Class to send a message by AWS SES."""
+
+    def __init__(self, message: Message, sender: str, recipients: List[str]) -> None:
+        """Initiate class AwsSesSender.
+
+        Args:
+            message (Message): Message to send
+            sender (str): Email from
+            recipients (List[str]): Emails to
+        """
+        super().__init__(message)
+        self._ses_client = boto3.client("ses")
+        self._sender = sender
+        self._recipients = recipients
+        self.verified_recipients = []
+
+    @property
+    def verified_recipients(self) -> List[str]:
+        """Verified by AWS SES recipiens."""
+        return self._verified_recipients
+
+    @verified_recipients.setter
+    def verified_recipients(self, recepients: List[str]) -> None:
+        """Verified by AWS SES recipiens."""
+        self._verified_recipients = recepients
+
+    def _get_identities(self, next_token: str = None) -> List[str]:
```

Comment:
I would prefer to move this functionality into another lib - aws-service related (we have s3-Manager already, it looks like we can also have ses_manager module with 1 function is_email_verified)